lms/djangoapps/program_enrollments/api/v1/serializers.py

```diff
@@ -40,3 +41,25 @@ class Meta(object):
 
     def get_account_exists(self, obj):
         return bool(obj.user)
+
+
+class ProgramCourseEnrollmentListSerializer(serializers.Serializer):
+    """
+    Serializer for listing course enrollments in a program.
+    """
+    student_key = serializers.SerializerMethodField()
```

Comment:
#NIT instead of `MethodField`, I thought you could just do `source='external_user_key'`, no?

Reply:
We have to pluck it out of the associated program enrollment, so unfortunately that won't work.

Reply:
Ah, right. A small but important distinction.